src/server/pmix_server.c

```diff
@@ -2243,6 +2244,12 @@ static void _mdxcbfunc(int sd, short argc, void *cbdata)
         }
     }
 
+    // Lock down all of the namespaces
+    PMIX_LIST_FOREACH(nptr, &nslist, pmix_nspace_caddy_t) {
+        PMIX_GDS_ACQUIRE_NS_LOCK(rc, nptr->ns);
```

Comment:
We aren't checking this return code anywhere

Reply:
Please add this check

Reply:
Check added